arcade/examples/sound_speed_demo.py

```diff
@@ -0,0 +1,171 @@
+"""
+Starting Template
+
+Once you have learned how to use classes, you can begin your program with this
+template.
+
+If Python and Arcade are installed, this example can be run from the command
+line with:
+
+python -m arcade.examples.sound_demo
+
+The top button is to play a music track.
+The 3 rows of buttons are arranged such that the audio is panned in the
+direction of the button, and the volume increases as you go down the column.
+
+Left click a button to play a sound. 
+If a sound is playing right click to increase volume, middle click to decrease.
+"""
+
+import typing
+
+import arcade
+
+SCREEN_WIDTH = 800
+SCREEN_HEIGHT = 600
+SCREEN_TITLE = "Starting Template"
+BUTTON_SIZE = 30
+
+
+class SoundButton(arcade.SpriteSolidColor):
+    """Button, click-for-sound"""
+
+    def __init__(self, sound_file, speed, volume, center_x, center_y):
+        super().__init__(BUTTON_SIZE, BUTTON_SIZE, arcade.color.WHITE)
+        self.sound = arcade.Sound(sound_file)
+        self.speed = speed
+        self.volume = volume
+        self.center_x = center_x
+        self.center_y = center_y
+
+    def play(self):
+        """Play"""
+        self.sound.play(speed=self.speed, volume=self.volume)
+
+    def draw(self):
+        super().draw(self)
+
+
+class MyGame(arcade.Window):
+    """
+    Main application class.
+
+    NOTE: Go ahead and delete the methods you don't need.
```

Comment:
I would delete this comment, as well as the methods that are stubbed out with `pass`.